.github/workflows/build.yml

```diff
@@ -118,7 +122,12 @@ jobs:
         if: github.event_name == 'push'
         run: |
           LIFECYCLE_IMAGE_TAG=$(git describe --always --dirty)
+          DOCKER_CLI_EXPERIMENTAL=enabled
           go run ./tools/image/main.go -lifecyclePath ./out/lifecycle-v*+linux.x86-64.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_IMAGE_TAG}-linux
+          go run ./tools/image/main.go -lifecyclePath ./out/lifecycle-v*+linux.arm64.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_IMAGE_TAG}-linux-arm64
```

Comment:
I think we'll need further changes to `tools/image/main.go` in order to produce an arm64 image - I'm guessing we'll need to pass `remote.WithDefaultPlatform` [here](https://github.com/buildpacks/lifecycle/blob/f15f80c6731ca9b01545eb8234281a105d3f4577/tools/image/main.go#L94) - though I'm not 100% sure how that will play with `remote.FromBaseImage`.

Reply:
It might make sense for "lifecycle image support" to be a separate PR...

Reply:
Are binaries independently useful outside of being packaged as images? I was hoping to produce an image with this PR that could be evaluated in more depth to flush out bugs before a release.

I've added an `-arch` flag to `tools/image/main.go`, and confirmed that the image's config is correctly set when pushing remotely.